js/latoken.js

```diff
@@ -0,0 +1,748 @@
+        let commision = parseFloat (trade['commision']);
+        let side = this.safeString(trade,'side');
+        let price = this.safeFloat(trade, 'price');
+        let amount = this.safeFloat(trade, 'amount');
+        let time = this.safeValue(trade, 'timestamp');
+        return {
+            'id': id,
+            'orderId': orderId,
+            'commision': commision,
+            'side': side,
+            'price': price,
+            'amount': amount,
+            'time': time,
+        }
+    }
+
+    parseOrderTrades (trades) {
+        let res = [];
+        for (let i = 0; i < trades.length; i++) {
+           res.push(this.parseOneOrderTrade(trades[i])); 
+        }
+        return res;
+    }
+
+    parseOrderTrade (trades, tradeKey = 'trades') {
+        return {
+            'pairId': trades['pairId'],
+            'symbol': trades['symbol'],
+            'tradeCount': trades['tradeCount'],
+            'trades': (tradeKey in trades) ? this.parseOrderTrades (trades[tradeKey]) : null,
+        }
+    }
+
+    async fetchMyTrades (symbol = undefined, params = {}, limit = 10) {
+        await this.loadMarkets ();
+        let request = {
+            'symbol': symbol,
+            'limit': limit
+        };
+        let response = await this.privateGetOrderTrades (this.extend(request, params));
+        return this.parseOrderTrade(response);
+    }
+
+    parseOrderStatus (status) {
+        let statuses = {
+            'active': 'open',
+            'partially_filled': 'open',
+            'filled': 'closed',
+            'canceled': 'canceled',
+        };
+        return (status in statuses) ? statuses[status] : status;
+    }
+
+    parseOrder (response) {
+        let orderId = response['orderId'];
+        let cliOrdId = response['cliOrdId'];
+        let pairId = this.safeValue(response, 'pairId');
+        let symbol = response['symbol'];
+        let side = response['side'];
+        let orderType = response['orderType'];
+        let price = this.safeFloat(response, 'price');
+        let amount = this.safeFloat(response, 'amount');
+        let orderStatus = response['orderStatus'];
+        let executedAmount = this.safeFloat(response, 'executedAmount');
+        let reaminingAmount = this.safeFloat(response, 'reaminingAmount');
+        let timeCreated = this.safeValue(response, 'timeCreated');
+        let timeFilled = (this.safeValue(response, 'timeFilled') == undefined) ? null : this.safeValue(response, 'timeFilled');
+        return {
+            'orderId': orderId,
+            'cliOrdId': cliOrdId,
+            'pairId': pairId,
+            'symbol': symbol,
+            'side': side,
+            'orderType': orderType,
+            'price': price,
+            'amount': amount,
+            'orderStatus': orderStatus,
+            'executedAmount': executedAmount,
+            'reaminingAmount': reaminingAmount,
+            'timeCreated': timeCreated,
+            'timeFilled': timeFilled,
```

Comment:
We need to make sure all structures returned from `fetch*` methods are unified according to https://github.com/ccxt/ccxt/wiki/Requirements